src/System.Device.Gpio/Interop/Unix/libgpiod/Interop.libgpiod.cs

```diff
@@ -0,0 +1,139 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information
+
+using System;
+using System.Device.Gpio;
+using System.Runtime.InteropServices;
+
+internal partial class Interop
```

Comment:
Are we missing a namespace?

Reply:
i don't really know why, but all partial classes for Interop are haven't set namespace. myabe @joperezr can explain

Reply:
I think it's fine if we follow existing pattern